### DISCUSSION TITLE
How can I prevent Obsidian's Quick Add from capturing a selection?

### DISCUSSION THREAD

**Discussion by qiaoxingxing** [chhoumann/quickadd#585] [Q&A]
Quick Add captures selected text, but I'd prefer to input text directly.
How can I prevent Quick Add from silently capturing selections?

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

This is configurable now (since QuickAdd 2.10.0). In Settings -> QuickAdd, under **Input**, turn off **Use editor selection as default Capture value** and Capture will always prompt you for input instead of silently grabbing the selection.

You can also override it per Capture choice with **Use editor selection as default value** (Follow global setting / Use selection / Ignore selection). The `{{SELECTED}}` token is unaffected, so you can still pull the selection in explicitly when you do want it.

Docs: https://quickadd.obsidian.guide/docs/Settings

Closing as resolved.
